Makefile

```diff
@@ -63,7 +64,10 @@ release-major release-minor release-patch:
 
 .PHONY: setup
 setup:
-	$(NPM) install
+	$(YARN)
+
+yarn.lock: package.json
+	$(YARN)
```

Comment:
The canonical usage is `yarn`,  they only have `install` for familiarity with npm.